simulation_ws/src/aws_robomaker_simulation_common/nodes/route_manager

```diff
@@ -1,305 +0,0 @@
-
-
-    def ravel_index(self, x, y):
-        '''
-        - description:
-            ravel 2d grid coordinates in row-major order   
-        - input: 
-            - ints
-                - x, y (in grid coordinates)
-        - output: int
-        '''
-        return y*(self.meta_data.width) + x
-
-
-    def grid_to_world_2d(self, x, y):
-        '''
-        - description:
-            transform x-y planar grid coordinates to world coordinates
-            adheres to the assumption that grid-world transform is only x-y translation and yaw rotation   
-        - input: 
-            - int, int
-                - x, y (in grid coordinates)
-        - output: [int, int]
-                x_world, y_world (in world coordinates)
-        '''
-        x_world = self.map_origin_x0 + (cos(self.map_yaw)*(self.resolution*x) - sin(self.map_yaw)*(self.resolution*y))
-        y_world = self.map_origin_y0 + (sin(self.map_yaw)*(self.resolution*x) + cos(self.map_yaw)*(self.resolution*y))
-
-        return [x_world, y_world]
-
-
-    def _create_pos(self, x_world, y_world, z_world, euler_orientation_x, euler_orientation_y, euler_orientation_z):
-        '''
-        - description:
-            wrap 3D euler location and orientation input to a Pose  
-        - input: 
-            - x, y, z (in world coordinates)
-            - orientation_x, orientation_y, orientation_z (in radians)
-        - output: 
-                format 
-                    - pose: 
-                          position: 
-                            x: double
-                            y: double
-                            z: double
-                          orientation:
-                            x: double
-                            y: double
-                            z: double
-                            w: double
-        '''
-        position = {
-                    'x':x_world,
-                    'y':y_world,
-                    'z':z_world
-                    }
-        
-        # Make sure the quaternion is valid and normalized
-        quaternion_orientation = transform.quaternion_from_euler(euler_orientation_x, euler_orientation_y, euler_orientation_z)
-        orientation = {
-                        'x':quaternion_orientation[0],
-                        'y':quaternion_orientation[1],
-                        'z':quaternion_orientation[2],
-                        'w':quaternion_orientation[3]
-                      }
-
-        p = {
-                'pose':
-                    {
-                        'position':position,
-                        'orientation':orientation
-                    }
-            }
-
-        return p
-
-
-    def check_noise(self, x, y, row_id=None):
-        '''
-        - description:
-            Low resolution/ noisy sensor data might lead to noisy patches in the map.
-            This function checks if the random valid point is not a noisy bleap on the map by looking for its neighbor consistency. 
```

Comment:
Let's keep the more detailed comments. Reformat it as necessary.

Reply:
I have added a concise yet equally descriptive description in place of this.